docs/managing-templates.md

```diff
@@ -134,3 +134,154 @@ Now let's make use of our new template.
         }
     ]
     ```
+
+## Adding a required field
+
+In may cases, we want users of the template to provide more detail, such as
+data that can be used to tailor the notifications based on the target at
+hand. This is accomplished via a [form fields](declarative-templates.md#example-form-fields).
+
+Let's make a new template that enables notifications via [Azure Devops Work
+Items](notifications/ado.md), where we require the user to specify the Area
+Path that the work items OneFuzz will create.
+
+This example will demonstrate setting the following:
+* `Project` via the project name specified during job creation.
+* `Area Path` via the new required field.
+* `Iteration Path` via a new optional field, with a default value in the template.
+
+1. Enable support for declarative templates.
+    ```
+    onefuzz config --enable_feature job_templates
+    ```
+2. Save a copy of the template locally.
+    ```
+    onefuzz job_templates manage get libfuzzer_linux > libfuzzer_linux_ado_areapath.json
+    ```
+3. With your preferred text editor, add the following to the `notifications` list:
+    ```json 
+    {
+      "config": {
+        "base_url": "https://dev.azure.com/org_name",
```

Comment:
```suggestion
        "base_url": "https://dev.azure.com/<your-org-name-here>",
```
Since `org_name` seems like it could be a generic subpath, for those not very familiar with ADO.